jetcd-all/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016-2021 The jetcd authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+plugins {
+    alias(libs.plugins.shadow)
+}
+
+dependencies {
+    api project(':jetcd-core')
+}
+
+shadowJar {
+    relocate 'io.grpc', 'io.etcd.jetcd.shaded.io.grpc'
+    relocate 'io.netty', 'io.etcd.jetcd.shaded.io.netty'
+    mergeServiceFiles()
+
+    archiveClassifier.set('')
```

Comment:
what is this about ?

Reply:
by default, the shadow jar will add the classifier `all` to the end. so the package name will be `jetcd-all-0.7.1-SNAPSHOT-all.jar`. This command will try to remove the classifier. So the name will be `jetcd-all-0.7.1-SNAPSHOT.jar` eventually.

Reply:
I thought since we are using a dedicated repo, we do not need the classifier now.